.ado/scripts/verdaccio.sh

```diff
@@ -14,9 +12,9 @@ case ${1-} in
 
   "init")
     npm set registry $NPM_REGISTRY
-    npx verdaccio --config $project_root/.ado/verdaccio/config.yaml &
```

Comment:
We don't need this line anymore?

Reply:
It can't be run inside a script, apparently. I had to move it back into the YAML file.